### ISSUE TITLE
inspect experienced an unexpected error during rpm -qf stage, if file missing

### ISSUE THREAD

**Issue by tigerfoot** [SUSE/machinery#1616]
-> Found 96 config files...Machinery experienced an unexpected error. Please file a bug report at: https://github.com/SUSE/machinery/issues/new
Execution of "ssh root@c-3po -o LogLevel\=ERROR LC_ALL\=en_US.utf8 rpm -qf /usr/share/vnc/classes/index.vnc\\\ \\\(replaced\\\)" failed with status 1: error: file /usr/share/vnc/classes/index.vnc (replaced): No such file or directory.
```
Error output:
error: file /usr/share/vnc/classes/index.vnc (replaced): No such file or directory

Backtrace:
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/bundle/ruby/2.0.0/gems/cheetah-0.4.0/lib/cheetah.rb:555:in `check_errors'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/bundle/ruby/2.0.0/gems/cheetah-0.4.0/lib/cheetah.rb:364:in `run'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/logged_cheetah.rb:23:in `run'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/remote_system.rb:84:in `run_command'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/rpm_database.rb:46:in `block in changed_files'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/rpm_database.rb:40:in `each_line'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/rpm_database.rb:40:in `each'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/rpm_database.rb:40:in `map'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/rpm_database.rb:40:in `changed_files'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/plugins/config_files/config_files_inspector.rb:51:in `inspect'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/inspect_task.rb:87:in `block in build_description'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/inspect_task.rb:79:in `each'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/inspect_task.rb:79:in `build_description'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/inspect_task.rb:21:in `inspect_system'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/lib/cli.rb:609:in `block (2 levels) in <class:Cli>'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/bundle/ruby/2.0.0/gems/gli-2.11.0/lib/gli/command_support.rb:126:in `call'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/bundle/ruby/2.0.0/gems/gli-2.11.0/lib/gli/command_support.rb:126:in `execute'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/bundle/ruby/2.0.0/gems/gli-2.11.0/lib/gli/app_support.rb:284:in `block in call_command'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/bundle/ruby/2.0.0/gems/gli-2.11.0/lib/gli/app_support.rb:297:in `call'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/bundle/ruby/2.0.0/gems/gli-2.11.0/lib/gli/app_support.rb:297:in `call_command'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/bundle/ruby/2.0.0/gems/gli-2.11.0/lib/gli/app_support.rb:79:in `run'
/usr/lib64/ruby/gems/2.0.0/gems/machinery-tool-1.16.0/bin/machinery:34:in `<top (required)>'
/usr/bin/machinery:24:in `load'
/usr/bin/machinery:24:in `<main>'
```
installed rpm from 
http://download.opensuse.org/repositories/systemsmanagement:/machinery/openSUSE_13.1/

On failing directory 
```
ll /usr/share/vnc/classes/
total 580
-rw-r--r-- 1 root root   2415 Oct 23  2013 AuthPanel.class
-rw-r--r-- 1 root root   2982 Oct 23  2013 ButtonPanel.class
-rw-r--r-- 1 root root   1176 Oct 23  2013 CapabilityInfo.class
-rw-r--r-- 1 root root   1888 Oct 23  2013 CapsContainer.class
-rw-r--r-- 1 root root   2591 Oct 23  2013 ClipboardFrame.class
-rw-r--r-- 1 root root   7822 Oct 23  2013 DesCipher.class
-rw-r--r-- 1 root root   2776 Oct 23  2013 InStream.class
-rw-r--r-- 1 root root    529 Oct 23  2013 MemInStream.class
-rw-r--r-- 1 root root   7066 Oct 23  2013 OptionsFrame.class
-rw-r--r-- 1 root root   6011 Oct 23  2013 RecordingFrame.class
-rw-r--r-- 1 root root   1405 Oct 23  2013 ReloginPanel.class
-rw-r--r-- 1 root root  20536 Oct 23  2013 RfbProto.class
-rw-r--r-- 1 root root   2646 Oct 23  2013 SessionRecorder.class
-rw-r--r-- 1 root root    317 Oct 23  2013 SocketFactory.class
-rw-r--r-- 1 root root   1584 Oct 23  2013 VncCanvas2.class
-rw-r--r-- 1 root root  26719 Oct 23  2013 VncCanvas.class
-rw-r--r-- 1 root root  17895 Oct 23  2013 VncViewer.class
-rwxr-xr-x 1 root root 444153 Oct 28 20:38 VncViewer.jar
-rw-r--r-- 1 root root   1996 Oct 23  2013 ZlibInStream.class

rpm -qf /usr/share/vnc/classes/
 tightvnc-1.3.10-14.2.1.x86_64
 xorg-x11-Xvnc-1.5.0-89.2.x86_64
```

**Comment by tigerfoot** [comment]
Futher checks 
```
 rpm -ql tightvnc 
/etc/slp.reg.d
/etc/slp.reg.d/vnc.reg
/etc/sysconfig/SuSEfirewall2.d/services/vnc-httpd
/etc/sysconfig/SuSEfirewall2.d/services/vnc-server
/etc/tightvncserver.conf
/etc/xinetd.d/vnchttpd
/usr/bin/vnc_inetd_httpd
/usr/bin/vncconnect
/usr/bin/vncpasswd
/usr/bin/vncpasswd.arg
/usr/bin/vncserver
/usr/bin/vncviewer
/usr/share/doc/packages/tightvnc
/usr/share/doc/packages/tightvnc/ChangeLog
/usr/share/doc/packages/tightvnc/LICENCE.TXT
/usr/share/doc/packages/tightvnc/README
/usr/share/doc/packages/tightvnc/WhatsNew
/usr/share/man/man1/vncconnect.1.gz
/usr/share/man/man1/vncpasswd.1.gz
/usr/share/man/man1/vncserver.1.gz
/usr/share/man/man1/vncviewer.1.gz
/usr/share/vnc
/usr/share/vnc/classes
/usr/share/vnc/classes/AuthPanel.class
/usr/share/vnc/classes/ButtonPanel.class
/usr/share/vnc/classes/CapabilityInfo.class
/usr/share/vnc/classes/CapsContainer.class
/usr/share/vnc/classes/ClipboardFrame.class
/usr/share/vnc/classes/DesCipher.class
/usr/share/vnc/classes/InStream.class
/usr/share/vnc/classes/MemInStream.class
/usr/share/vnc/classes/OptionsFrame.class
/usr/share/vnc/classes/RecordingFrame.class
/usr/share/vnc/classes/ReloginPanel.class
/usr/share/vnc/classes/RfbProto.class
/usr/share/vnc/classes/SessionRecorder.class
/usr/share/vnc/classes/SocketFactory.class
/usr/share/vnc/classes/VncCanvas.class
/usr/share/vnc/classes/VncCanvas2.class
/usr/share/vnc/classes/VncViewer.class
/usr/share/vnc/classes/VncViewer.jar
/usr/share/vnc/classes/ZlibInStream.class
/usr/share/vnc/classes/index.vnc
bruno@c-3po:~$ rpm -ql xorg-x11-Xvnc
/etc/X11/xorg.conf.d/10-libvnc.conf
/etc/slp.reg.d
/etc/slp.reg.d/vnc.reg
/etc/sysconfig/SuSEfirewall2.d/services/vnc-httpd
/etc/sysconfig/SuSEfirewall2.d/services/vnc-server
/etc/vnc
/etc/vnc/tls.cert
/etc/vnc/tls.key
/etc/xinetd.d/vnc
/usr/bin/Xvnc
/usr/bin/vnc_inetd_httpd
/usr/bin/vncconfig
/usr/bin/vncpasswd
/usr/bin/vncpasswd.arg
/usr/bin/vncserver
/usr/bin/x0vncserver
/usr/lib64/xorg/modules/extensions/libvnc.so
/usr/share/doc/packages/xorg-x11-Xvnc
/usr/share/doc/packages/xorg-x11-Xvnc/LICENCE.TXT
/usr/share/doc/packages/xorg-x11-Xvnc/README
/usr/share/doc/packages/xorg-x11-Xvnc/README.txt
/usr/share/man/man1/Xvnc.1.gz
/usr/share/man/man1/vncconfig.1.gz
/usr/share/man/man1/vncpasswd.1.gz
/usr/share/man/man1/vncserver.1.gz
/usr/share/man/man1/x0vncserver.1.gz
/usr/share/vnc
/usr/share/vnc/classes
/usr/share/vnc/classes/VncViewer.jar
```

**Comment by tgoettlicher** [comment]
Thank you for reporting this issue.

It seems as if Machinery thinks the output `/usr/share/vnc/classes/index.vnc (replaced)` is a file name.

We will look into that...

**Comment by mauromorales** [comment]
@tigerfoot this is fixed in release 1.18.0
